generators/app/templates/webapp/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"_version": "1.34.0",
+	"_version": "1.0.0",
```

Comment:
This is indeed a good point. We need to make this version dependend: https://github.com/SAP/ui5-manifest/blob/master/mapping.json - I would not use `1.0.0` but for now it's ok - I'll prepare a change for the other generators and you can reuse it - maybe this is also something to discuss to create a `util-generator-ui5` modules which contains such kind of helpers...

Reply:
What else should be used? This should become te version of the app no? Or do you prefer 0.0.1 ?